components/widgets/resources/src/main/resources/resources/uicomponents/widgets/extraGeneVariantData.js

```diff
@@ -3,35 +3,37 @@ var ExtraGeneVariantData = (function (ExtraGeneVariantData) {
   tools.Editor = Class.create({
 
     initialize : function () {
-      var geneTableListId = $$('.gene-table.extradata-list')[0].id;
-      this.geneClassName = geneTableListId.substring(geneTableListId.lastIndexOf('-') + 1);
-      this.geneVariantClassName = 'PhenoTips.GeneVariantClass';
```

Comment:
This line was removed, and now `geneVariantClassName` is no longer defined anywhere, although it is still used in several places.